lib/get/onMissing.js

```diff
@@ -33,11 +28,13 @@ module.exports = function onMissing(model, path, depth,
 
 function concatAndInsertMissing(model, remainingPath, depth, requestedPath,
                                 optimizedPath, optimizedLength, results) {
-    results.requestedMissingPaths[results.requestedMissingPaths.length] =
-        fastCat(arraySlice(requestedPath, 0, depth), remainingPath);
```

Comment:
This creates a slice just to pass it to concat, resulting in an extra array allocation.